wandb/apis/public/runs.py

```diff
@@ -921,7 +921,10 @@ def metadata(self):
         if self._metadata is None:
             try:
                 f = self.file("wandb-metadata.json")
-                contents = util.download_file_into_memory(f.url, wandb.Api().api_key)
```

Comment:
can this utility be deleted?

Reply:
there's one more part where seem the utility being used: https://github.com/wandb/wandb/blob/2eaa9ae9f201a408c369f7f878c921e4ecbc3c43/wandb/apis/importers/wandb.py#L607

Reply:
but likely a similar approach could be used in that other place too - @dmitryduev should I look into it as part of this PR?